app/src/main/java/co/tinode/tindroid/media/AttachmentType.java

```diff
@@ -0,0 +1,19 @@
+package co.tinode.tindroid.media;
+
+/**
+ * Attachment types
+ */
+
+public class AttachmentType {
```

Comment:
If you define constants, why not use `Enum` instead of static class members?

Reply:
It's just habit, I will use `Enum`